src/public/client.ts

```diff
@@ -220,3 +223,27 @@ fetch('/api/availableDomains')
     selectedHost = domains[0]
     hostSelector.innerText = domains[0]
   })
+
+fetch('/api/status')
+  .then(r => r.json())
+  .then(data => {
+    const dataObj = JSON.parse(data.stdout).map(el => ({
+      name: el.name,
+      status: el.pm2_env.status
+    }))
+
+    dataObj.forEach(el => {
+      const statusContainer = document.getElementById(el.name)
```

Comment:
let's make this more modular by putting the status logic inside mapping item. Current implementation feels a little hacky and becomes difficult to maintain. Logically, MappingItem class should take care of everything relating to mapping (including status). To do this, change the code above from:
```
fetch('/api/mappings')
 .then(r => r.json())
 .then((data: Mapping[]) => {....
```

To:
```
Promise.all([
  fetch('/api/mappings').then( r => r.json()),
  fetch('/api/statuses').then( r => r.json())
]).then( ([mappings: Mapping[], statuses: Status[]]) => {
  const fullDomainStatusMap = statuses.reduce(....., {})
  domainList.innerHTML = ''
  mappings
    .reverse()
    .filter(e => e.domain && e.port && e.id && e.ip && e.gitLink && e.fullDomain)
   .forEach( mapping => {
      new MappingItem(mapping, fullDomainStatusMap[ mapping.fullDomain ] )
  })
})
```

Then update `MappingItem` class to draw directly into render (you will no longer need `id` for tag attribute).